From 1eaac3cd2b3a036f5d40a646d59622345d2dab44 Mon Sep 17 00:00:00 2001
From: Adhemerval Zanella <adhemerval.zanella@linaro.org>
Date: Fri, 25 Mar 2022 11:26:26 -0300
Subject: [PATCH 032/163] resolv: Initialize loop variable on
 tst-resolv-trailing

Checked on x86_64-linux-gnu and i686-linux-gnu.

(cherry picked from commit 2fe316888a5d4ef9b215c7de2532ce981930b97e)
---
 resolv/tst-resolv-trailing.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/resolv/tst-resolv-trailing.c b/resolv/tst-resolv-trailing.c
index c42e2a34d6..ce141759ad 100644
--- a/resolv/tst-resolv-trailing.c
+++ b/resolv/tst-resolv-trailing.c
@@ -114,7 +114,7 @@ do_test (void)
          AI_V4MAPPED,
          AI_V4MAPPED | AI_NUMERICHOST,
         };
-      for (size_t gai_flags_idx; gai_flags_idx < array_length (gai_flags);
+      for (size_t gai_flags_idx = 0; gai_flags_idx < array_length (gai_flags);
              ++gai_flags_idx)
         {
           struct addrinfo hints = { .ai_flags = gai_flags[gai_flags_idx], };
-- 
2.37.2

